ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -650,16 +653,7 @@ export default class ConfirmAmendment extends Vue {
     }
 
     // get registration data from api and load into store
-    try {
-      await this.loadRegistration()
-    } catch (error) {
-      const errorMsg = error as string
-      console.error(errorMsg)
-      this.emitError({
-        statusCode: 500,
-        message: errorMsg
-      })
-    }
+    await this.loadRegistration()
```

Comment:
do you want to handle any error returned from here?

Reply:
it's already handled within the fn